docs/guides/react-native.md

```diff
@@ -1,6 +1,6 @@
 # Using Enzyme to Test Components in React Native
 
-As of React 0.18, React Native uses React as a dependency rather than a forked version of the library,
+As of React Native 0.18, React Native uses React as a dependency rather than a forked version of the library,
```

Comment:
Maybe instead we can say "As of version 0.18, React Native" or just "As of  0.18, React Native"? It sounds weird having "React Native" repeat within one word.